third_party/move/move-compiler-v2/transactional-tests/tests/misc/bug_14243_stack_size.baseline.exp

```diff
@@ -1,32 +1,33 @@
-processed 1 task
-
-task 0 'print-bytecode'. lines 2-14:
-// Move bytecode v7
-module c0ffee.m {
-
-
-id_mut<Ty0>(Arg0: &mut Ty0): &mut Ty0 /* def_idx: 0 */ {
-B0:
-	0: MoveLoc[0](Arg0: &mut Ty0)
-	1: Ret
-}
-t0() /* def_idx: 1 */ {
-L0:	loc0: u64
-L1:	loc1: &mut u64
-B0:
-	0: LdU64(0)
-	1: StLoc[0](loc0: u64)
-	2: MutBorrowLoc[0](loc0: u64)
-	3: StLoc[1](loc1: &mut u64)
-	4: CopyLoc[1](loc1: &mut u64)
-	5: Call id_mut<u64>(&mut u64): &mut u64
-	6: ReadRef
-	7: Pop
-	8: MoveLoc[1](loc1: &mut u64)
-	9: ReadRef
-	10: Pop
-	11: Ret
-}
-}
-
-==> Compiler v2 delivered same results!
+comparison between v1 and v2 failed:
+= processed 1 task
+= 
+= task 0 'print-bytecode'. lines 2-14:
+- // Move bytecode v7
++ // Move bytecode v8
```

Comment:
Seems that V1 compiler is still on 7?

Reply:
Diff of a diff between v1 and v2, yes, it still is.